src/model.coffee

```diff
@@ -82,7 +82,7 @@ class Model
     # Returns null
     @save: (id, data, callback) ->
         @adapter.save id, data, (err, attributes) =>
```

Comment:
Oh that's why I didn't understand…it seems a bit overzealous to me to do in two lines what is straightforward in one, especially since the purpose of this way of handling errors is to prevent conditional nesting, which is absent here. I'm doing the change though.